src/__tests__/App.test.jsx

```diff
@@ -29,23 +41,44 @@ describe("App component", () => {
     // expect(screen.getByLabelText("name-input")).toBeDefined();
 
   });
-
+  
   it("allows the user to type their name", () => {
-    const testComponent = (
-      <App />
-    );
-    render(testComponent);
-    const inputField = screen.getByPlaceholderText("name");
-
-    fireEvent.change(inputField, {target: {value: 'Test'}})
-
-    const inputtedName = inputField.value
-    expect(inputtedName).toBe("Test"); 
-  });
+      const testComponent = (
+          <App />
+          );
+          render(testComponent);
+          const inputField = screen.getByPlaceholderText("name");
+          
+          fireEvent.change(inputField, {target: {value: 'Test'}})
+          
+          const inputtedName = inputField.value
+          expect(inputtedName).toBe("Test"); 
+        });
+
+
+    it("prompts the user to input the month", () => {
+        render(<App />);
+        expect(screen.getByText("Month:")).toBeDefined()
```

Comment:
I like how concise these tests are! Could you rewrite the ones above to match this format?